unittests/InstructionCountCI/SecondaryGroup.json

```diff
@@ -1640,19 +1639,19 @@
         "str q31, [x4, #400]",
         "ubfx x20, x4, #1, #2",
         "cbnz x20, #+0x8",
-        "b #+0x2c",
+        "b #+0x28",
         "mov w20, #0x1f80",
         "mrs x21, fpcr",
         "ubfx x21, x21, #22, #3",
         "rbit w0, w21",
         "bfi x21, x0, #30, #2",
         "bfi w20, w21, #13, #3",
-        "add x21, x4, #0x18 (24)",
         "str w20, [x4, #24]",
         "mov w20, #0xffff",
-        "str w20, [x21, #4]",
-        "ubfx x20, x4, #0, #3",
-        "str x20, [x4, #512]"
+        "str w20, [x4, #28]",
+        "add x20, x4, #0x200 (512)",
+        "ubfx x21, x4, #0, #3",
+        "str x21, [x20]"
```

Comment:
This is an interesting change. Should be able to still do `str x21, [x4, #512]`

Reply:
Nice, fixed.